graylog2-server/src/main/java/org/graylog2/indexer/indices/Indices.java

```diff
@@ -106,11 +108,18 @@ public void move(String source, String target) {
     }
 
     public void delete(String indexName) {
+        Optional<SearchableSnapshotInfo> snapshotInfoOptional = indicesAdapter.getSearchableSnapshotInfo(indexName);
         indicesAdapter.delete(indexName);
+
         eventBus.post(IndicesDeletedEvent.create(indexName));
+        snapshotInfoOptional.ifPresent(snapshotInfo -> eventBus.post(new DeleteSnapshotEvent(snapshotInfo)));
     }
 
     public void close(String indexName) {
+        indicesAdapter.getSearchableSnapshotInfo(indexName).ifPresent(snapshotInfo -> {
+            throw new UnsupportedOperationException("Close operation not available for warm index: " + snapshotInfo.currentIndexName());
```

Comment:
I think that's fine here, but we still need to somehow let the frontend know that warm tier indices are not closeable and to hide the close button for them.

Reply:
we can do that later, not important atm